.github/workflows/deploy.yml

```diff
@@ -0,0 +1,149 @@
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_SES_SOURCE_ARN: ${{ secrets.AWS_SES_SOURCE_ARN }}
+      OKTA_TEST_PASSWORD: ${{ secrets.OKTA_TEST_PASSWORD }}
+      OKTA_TEST_SECRET: ${{ secrets.OKTA_TEST_SECRET }}
+      OKTA_TEST_USERNAME: ${{ secrets.OKTA_TEST_USERNAME }}
+      AWS_ROLE_TO_ASSUME: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+      STORYBOOK_S3_BUCKET_DEV: ${{ secrets.STORYBOOK_S3_BUCKET_DEV }}
+
+  client_typecheck:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      issues: write
+    steps:
+      - name: Check out code
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2.3.4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Set up node
+        uses: actions/setup-node@9ced9a43a244f3ac94f13bfd896db8c8f30da67a # v2.1.5
+        with:
+          node-version: ${{ env.EASI_APP_NODE_VERSION }}
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - name: Configure yarn cache
+        uses: actions/cache@937d24475381cd9c75ae6db12cb4e79714b926ed # v2.1.6
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: |
+            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+            ./node_modules
+          key: v1-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            v1-yarn-
+      - name: Run typechecking
+        id: typechecker
+        # Uses GitHub environment files to store typechecker output and pass to next stage
+        # see https://github.com/actions/toolkit/blob/main/docs/commands.md#environment-files
+        run: |
+          yarn build:ts 2>&1 | tee test.log
+          result_code=${PIPESTATUS[0]}
+          echo 'TS_LOG<<EOF' >> $GITHUB_ENV
+          cat test.log >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+          exit $result_code
+        continue-on-error: true
+      - name: Post comment with TypeScript type errors
+        uses: actions/github-script@v6
+        if: ${{ steps.typechecker.outcome == 'failure' }}
+        with:
+          script: |
+            const typeScriptLog = process.env.TS_LOG;
+
+            const commentBody = `
+            <details>
+            <summary>TypeScript compile errors</summary>
+
+            \`\`\`shell
+            ${typeScriptLog}
+            \`\`\`
+            </details>`;
+
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: commentBody
+            });
```

Comment:
I think we should still remove this step of the job, since `deploy.yml` runs on pushes to `master`, there will not be a `context.issue.number`, and this will fail (You can see an example of this [here](https://github.com/CMSgov/easi-app/runs/5587211988?check_suite_focus=true))

Reply:
Yes, this is the point I was trying to make in the other comment thread.

Reply:
Resolved with https://github.com/CMSgov/easi-app/pull/1537/commits/dae7a6e8a55d3decc9824e7654ef04a73949a668.